src/app/clusters/access-control-server/access-control-server.cpp

```diff
@@ -296,7 +296,7 @@ CHIP_ERROR AccessControlAttribute::WriteExtension(const ConcreteDataAttributePat
         ReturnErrorOnFailure(aDecoder.Decode(list));
 
         size_t count = 0;
-        ReturnErrorOnFailure(list.ComputeSize(&count));
+        ReturnErrorOnFailure(list.ComputeSize(count));
```

Comment:
This is a breaking change to a public API that _will_ break SDK consumers.  The bar for doing this sort of thing is very high.

Reply:
This one is easy to fix (fixed)